recipe/patches/0003-Ignore-warnings-in-event.cc-and-logging.cc.patch

```diff
@@ -1,14 +1,14 @@
-From db16dd7d8151223cbaaf8b283560503201356094 Mon Sep 17 00:00:00 2001
-From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Fri, 3 Dec 2021 10:59:10 -0800
-Subject: [PATCH 03/10] Ignore warnings in event.cc and logging.cc
+From fbc686592a155fcb8dba463497b61ad24207eea3 Mon Sep 17 00:00:00 2001
+From: Matti Picus <matti.picus@gmail.com>
+Date: Sun, 19 Jan 2025 22:56:08 +1100
+Subject: [PATCH] Ignore warnings in event.cc and logging.cc
```

Comment:
You should try to keep author and date the same.
https://github.com/conda-forge/ray-packages-feedstock/blob/4bb7180a1c88a89a8b5777ee1a191be97c212afb/ray-patching.md?plain=1#L39-L45

Of course, for rebasing and other changes you can add a `Co-Authored-By: Matti Picus <matti.picus@gmail.com>` line to the commit message.

Reply:
Thanks, will revert

Reply:
reviewed attributions in 9796fdb